docs/conf.py

```diff
@@ -14,29 +14,33 @@
 #
 import os
 import sys
-sys.path.insert(0, os.path.abspath('../'))
-sys.path.insert(0, os.path.abspath('../stravalib'))
+
+sys.path.insert(0, os.path.abspath("../"))
+sys.path.insert(0, os.path.abspath("../stravalib"))
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-#sys.path.insert(0, os.path.abspath('.'))
+# sys.path.insert(0, os.path.abspath('.'))
 
 # -- General configuration ------------------------------------------------
 
 # -- Project information -----------------------------------------------------
 
 import datetime
-import stravalib 
+
+import stravalib
 
 # General project info
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
-project = 'stravalib'
+project = "stravalib"
```

Comment:
i ran black on the conf.py file because i had the pre-comit hook running locally so it ran when i committed the change.